ui/raidboss/data/04-sb/dungeon/doma_castle.txt

```diff
@@ -0,0 +1,137 @@
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839:.*is no longer sealed/ window 5000 jump 0
+
+### Magitek Rearguard 
+# -ii 209F 20A0
+0 "Start" sync / 00:0839:The Third Armory will be sealed off/
+7.4 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+15.0 "Garlean Fire" sync /:Magitek Rearguard:209E:/
+30.4 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+35.4 "Mines" sync /03:........:Added new combatant Rearguard Mine/
+37.6 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+40.6 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+43.2 "Garlean Fire" sync /:Magitek Rearguard:209E:/
+48.7 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+53.7 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+60.7 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+
+68.3 "Mines" sync /03:........:Added new combatant Rearguard Mine/
+70.5 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+73.5 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+76.2 "Garlean Fire" sync /:Magitek Rearguard:209E:/
+81.7 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+85.8 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+88.7 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+91.4 "Garlean Fire" sync /:Magitek Rearguard:209E:/
+97.0 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+98.5 "Mines" sync /03:........:Added new combatant Rearguard Mine/
+101.1 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+104.1 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+106.8 "Garlean Fire" sync /:Magitek Rearguard:209E:/
+112.3 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+115.4 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+118.4 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+
+131.1 "Mines" sync /03:........:Added new combatant Rearguard Mine/ jump 68.3
+132.0 "Cermet Pile"
```

Comment:
The difference between Mines and Cermet Pile at 68.3 is 2.2 seconds, but is 0.9 seconds at 131.1.  Is `Cermet Pile` the loop and not `Mines`? What does `test_timeline` say about the amount it jumps when you run it against several logs?

Also, this it the tiniest of code review nits, but for loops can you run `node --loader ts-node/esm util/timeline_adjust.mjs -f ui/raidboss/data/04-sb/dungeon/doma_castle.txt -a 62.8` and then use the times from `68.3` that have been moved forward 62.8 seconds (131.1 - 68.3) in time so that the loop lookahead has consistent times with where it's jumping?  In the future I'd like to just add a `loop` command to timeline so that loops don't have to be manual, but we're not there yet.